### DISCUSSION TITLE
ebusd V5 Http Get Request

### DISCUSSION THREAD

**Discussion by physman38** [john30/ebusd#1207] [Q&A]
Dear @john30 
Dear all,

I have an issue and need some help from you.
The EBUSD V5 adapter is working nice so far. Now I would need to do HTTP GET WEBRequests for my Gira X1.
So I have configured the ebusd on a RaspPi and gave the ebsud file the following line:
`EBUSD_OPTS="--configpath=/etc/ebusd/ebusd-configuration/ebusd-2.1.x/de --scanconfig=full -d enh:xx.xx.xx.xx:xxxx --pollinterval=10 --accesslevel=* --mqtthost=xx.xx.xx.xx --mqttport=yyyy --mqttuser=USER --mqttpass=PASS --httpport=AAAA --mqttjson"
`
As the GIRA X1 seems to have some problems with 1 LogikEngine on MQTT, I would like to switch to HTTP.

So if I enter in a WebBrowser eg:
`http://xxx.xxx.xxx.xxx:yyyy/data/bai/solpump`
I get the response with no issue.

If use a WebRequest from the GIRA X1, in Simulation Mode it gets the requests (eg I request every 10sec).
If I leave the Simulation, and put it to active, ebusd is not updating the info from the HTTP.

I have added "r2" to the respective CSV files.  I have put the "pollinterval" as above.

Can you imagine what I do wrong here?

Thanks so much for your help!

**Comment by john30** [comment]
you'd have to intercept the traffic in order to find out. for a start, you could easily use netcat to simulate the ebusd endpoint e.g. by issuing "nc -l 8080" on the host where ebusd normalle runs (ebusd needs to be stopped during that simulation or http port switched off).
then the request data should appear on the command line.
hm, just wondering if you mixed up the mqtt port with the http port? as you 'yyyy' only appear as the mqttport argument to ebusd
